### ISSUE TITLE
i am getting undefined value

### ISSUE THREAD

**Issue by harishmahajan** [kosiakMD/react-global-storage#1]
Meteor.call('settings.insert',{generalData},function(err,res){
                globalStorage.setState('settingid',res._id);
                let value2 = globalStorage.state.settingid;
            });
                console.log("id:- ",value2);
when i use above code then i am getting 'undefined' for value2

==============================================
            Meteor.call('settings.insert',{generalData},function(err,res){
                globalStorage.setState('settingid',res._id);
                let value2 = globalStorage.state.settingid;
                console.log("id:- ",value2);
            });

when i use above code then i got the actual id. 

in short, globalstorage not works. 

what is the meaning of global?
==> you can access it anywhere.
